src/SwedbankPay/Core/Adapter/WC_Adapter.php

```diff
@@ -549,7 +549,14 @@ public function savePaymentToken(
         $expiry_date = explode('/', $expiry_date);
 
         // Create Payment Token
-        $token = new WC_Payment_Token_Swedbank_Pay();
+        if (!property_exists($this->gateway, 'payment_token_class') ||
+            empty($this->gateway->payment_token_class)
+        ) {
+            throw new \Exception(__('Payment Token class is undefined.', 'swedbank-pay-woocommerce-payments'));
```

Comment:
Will this now require that WooCommerce Subscriptions is installed, or under which circumstances is `$this->gateway->payment_token_class` set?

Reply:
@asbjornu No, It won't reqture WC Subscription. That's like Vault of Magento 2. WooCommerce use it for handling of recurring tokens